scripts/jenkins-build.sh

```diff
@@ -109,8 +115,8 @@ else
 fi
 # need to call first build
 
-if [[ -d Wrappers/Python/conda-recipe ]]; then
-  eval conda build Wrappers/Python/conda-recipe "$CCPI_BUILD_ARGS" "$@"
+if [[ -d ${RECIPE_PATH} ]]; then
```

Comment:
This will force us to change all Jenkins configurations.
Could you do
```bash

if [[ ! -n ${RECIPE_PATH} ]] then
  export RECIPE_PATH=Wrappers/Python/conda-recipe
fi
```
This way we can always use the `RECIPE_PATH` with a default written in the build script.

Reply:
I have done that already in lines 38-40. Maybe I should move them further down to make it clearer